ankdown/ankdown.py

```diff
@@ -210,11 +213,18 @@ def __getitem__(self, deckname):
         return super(DeckCollection, self).__getitem__(deckname)
 
 def field_to_html(field):
-    # Need to extract the math in brackets so that it doesn't get
-    # markdowned.
-    for bracket in ["(", ")", "[", "]"]:
-        field = field.replace(r"\{}".format(bracket), r"\\{}".format(bracket))
-        # backslashes, man.
+    """Need to extract the math in brackets so that it doesn't get markdowned.
+    If math is separated with dollar sign it is converted to brackets."""
+    if FLAG_DOLLAR:
+        for sep_dollar, sep_bracket in zip(["$", "$$"], [[r"\\(", r"\\)"], [r"\\[", r"\\]"]]):
+            field = field.split(sep_dollar)
```

Comment:
since the single dollar signs are processed first, won't this never actually process `$$`?

Reply:
Oh this is a big issue! By changing the order we can avoid that.